### ISSUE TITLE
Running 'bundle install' gives invalid gemspec warning

### ISSUE THREAD

**Issue by nitinstp23** [mattmueller/foursquare2#90]
Running `bundle install` gives this warnings - 

```
Installing hashie 3.4.0

foursquare2 at /home/nitin/work/foursquare2 did not have a valid gemspec.
This prevents bundler from installing bins or native extensions, but that may not affect its functionality.
The validation message from Rubygems was:
  ["Gemfile.lock"] are not files
Using foursquare2 2.0.1 from source at .
```

**Comment by kalashnikovisme** [comment]
@nitinstp23 use your repo, while it's not merged. Thanks, man:)

**Comment by nitinstp23** [comment]
:+1:

**Comment by mattmueller** [comment]
Hey all - 

I've merged in the fix and tried to do a bit of cleanup. As part of this I've released 2.02 to rubygems - let me know if you have any issues.

Sorry for the slow response.
